tests/python/rest_api/test_requests.py

```diff
@@ -0,0 +1,314 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import io
+from http import HTTPStatus
+from typing import Callable, List
+from urllib.parse import urlparse
+
+import pytest
+from cvat_sdk.api_client import ApiClient, models
+
+from shared.utils.config import make_api_client
+from shared.utils.helpers import generate_image_files
+
+from .utils import (
+    create_task,
+    export_job_dataset,
+    export_project_backup,
+    export_project_dataset,
+    export_task_backup,
+    export_task_dataset,
+    import_task_backup,
+)
+
+
+def validate_action(filter_: str, filter_value: str, requests: List[models.Request]) -> bool:
+    return {filter_value} == {r.operation.type.split(":")[0] for r in requests}
+
+
+def validate_subresource(filter_: str, filter_value: str, requests: List[models.Request]) -> bool:
+    return {filter_value} == {r.operation.type.split(":")[1] for r in requests}
+
+
+def validate_status(filter_: str, filter_value: str, requests: List[models.Request]) -> bool:
+    return {filter_value} == {r.status.value for r in requests}
+
+
+def validate_id(filter_: str, filter_value: int, requests: List[models.Request]) -> bool:
+    return {filter_value} == {getattr(r.operation, filter_) for r in requests}
+
+
+def validate_resource(filter_: str, filter_value: str, requests: List[models.Request]) -> bool:
+    return {filter_value} == {r.operation.target.value for r in requests}
+
+
+def validate_format(filter_: str, filter_value: str, requests: List[models.Request]) -> bool:
+    return {filter_value} == {r.operation.format for r in requests}
```

Comment:
Probably, these functions are supposed to be class members in the list testing class.

Reply:
Replaced this logic by using `CollectionSimpleFilterTestBase`